neurons/miner.py

```diff
@@ -32,19 +33,51 @@
 from scraping.coordinator import ScraperCoordinator
 from scraping.provider import ScraperProvider
 from storage.miner.sqlite_miner_storage import SqliteMinerStorage
+from neurons.config import NeuronType, check_config, create_config
 
-from neurons.base_neuron import BaseNeuron
 
-
-class Miner(BaseNeuron):
+class Miner:
     """The Glorious Miner."""
 
     def __init__(self, config=None):
-        super().__init__(config=config)
+        self.config = copy.deepcopy(config or create_config(NeuronType.MINER))
+        check_config(self.config)
+
+        bt.logging(config=self.config, logging_dir=self.config.full_path)
+        bt.logging.info(self.config)
 
         if self.config.offline:
-            bt.logging.success("Running in offline mode. Skipping axon creation.")
+            bt.logging.success(
+                "Running in offline mode. Skipping bittensor object setup and axon creation."
+            )
         else:
+            # The wallet holds the cryptographic key pairs for the miner.
+            self.wallet = bt.wallet(config=self.config)
+            bt.logging.info(f"Wallet: {self.wallet}.")
+
+            # The subtensor is our connection to the Bittensor blockchain.
+            self.subtensor = bt.subtensor(config=self.config)
```

Comment:
Can you double check if we still need this? I _think_ it may only be used by the "block()" method, which I propose we get rid of anyway.

Reply:
Per discussion we still use this on resyncing the metagraph although I have removed the block based checks.